deploy/cert-manager-webhook-pdns/values.yaml

```
@@ -58,3 +58,5 @@ containerSecurityContext:
 podAnnotations: {}
 
 podSecurityContext: {}
+
+priorityClassName: {}
```

@owngr I believe the default for this should be `priorityClassName: ""` since it's a string value and not an object. Would you mind updating that for me?

```suggestion
priorityClassName: ""
```

---

@zachomedia thanks for the suggestion it makes more sense indeed, I added it to the commit to have a single commit